#Request : expects position setpoint via x, y, z
#Request : expects yaw setpoint via yaw (send yaw_valid=true)
geometry_msgs/TwistStamped twist #deprecated, instead use x,y,z,yaw
float32 x
float32 y
float32 z
float32 yaw
float32 tolerance
bool async
bool relative
bool yaw_valid
bool body_frame
---
#Response : success=true - (if async=false && if setpoint reached before timeout = 30sec) || (if async=true)
bool success
string message